evergreen/run-mongodb-aws-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
A random side nit for this and other bash scripts: 
My loose understanding is that using
`#!/usr/bin/env bash`
increases the portability of the script (as bash may not always be in /bin/bash). Probably not an issue in this case since our scripts were tailored for Cygwin, where bash does indeed exist at `/bin/bash` all the time (AFAIK). So again, as this is merely a nit, feel free to ignore :)